### ISSUE TITLE
Data availability 

### ISSUE THREAD

**Issue by Minys233** [XericZephyr/seq2seq-fingerprint#58]
Great work guys! I'm re-implementing this seq2seq model, but somehow I'm facing issues about data used in the paper.

In the original paper, the seq2seq model is trained with the combination of two datasets: LogP and PM2-10k (the same data is used in your seq3seq paper later in 2018). However, in these two paper, no references could be found for the data source, and neither in the github repos. In addition, code in github suggests you trained your model with ZINC15, different from what you did in the paper.

So pls provide the original data source (LogP and PM2-10k) that corresponds to the results in the paper. Many thanks.

**Comment by JanoschMenke** [comment]
Yes I also could not find the datasets that are referenced in the two papers

**Comment by Minys233** [comment]
> Yes I also could not find the datasets that are referenced in the two papers

Two paper also amazingly share some similar figures & paragraphs, wonder if it is okey to do that.
